content/show/coder-radio/subscribe.md

```diff
@@ -1,6 +1,21 @@
 ---
 title: "Subscribe"
-date: 2022-08-16T05:32:13-04:00
+date: 2022-08-25
 draft: false
+rss: "https://coder.show/rss"
+direct_links:
+    apple: https://podcasts.apple.com/us/podcast/coder-radio/id534941512
+    google_podcast: https://podcasts.google.com/?feed=aHR0cHM6Ly9jb2Rlci5zaG93L3Jzcw==
+    castro: https://castro.fm/itunes/534941512
+    castbox: https://castbox.fm/channel/Coder-Radio-Video-id2482150
+    overcast: https://overcast.fm/itunes534941512/coder-radio-mp3
+    pocket_cast: https://pca.st/BCivrU
+    spotify: https://open.spotify.com/show/04WzuGq8oAcBI8GHlyvmI8?si=PHKUungkTi2SBQddVnQ8rw
+    stitcher: https://www.stitcher.com/podcast/jupiter-broadcasting/coder-radio-mp3
+    tunein: https://tunein.com/podcasts/Podcasts/Coder-Radio-p945854/
+    iheart_radio: https://www.iheart.com/podcast/256-coder-radio-31099184/
+    radio_public: https://radiopublic.com/coder-radio-GbeZx8
+    podcast_index: https://podcastindex.org/podcast/487548
+
 ---
```

Comment:
verified all of these links via local dev environment and they're all functional and correct. 👍🏻